src/cc/bpf_module.cc

```diff
@@ -95,6 +98,8 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags)
     : flags_(flags), ctx_(new LLVMContext) {
+  InitializeNativeTarget();
```

Comment:
ahh. ok. I hope both key and value will be possible to represent this way eventually, like
echo "{1 2}  {3 4}" > new_entry
where key and value are structs of two u32s.